test/tests/open-liberty-hello-world/run.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# since we have curl in the liberty image, we'll use that
+clientImage="$1"
+
+serverImage="$1"
+
+# Create an instance of the container-under-test
+cid="$(docker run -d "$serverImage")"
+trap "docker rm -vf $cid > /dev/null" EXIT
+
+_request() {
+	local url="${1#/}"
+	shift
+
+	docker run --rm --link "$cid":open-liberty "$clientImage" \
+		wget -q -O - "$@" "http://open-liberty:9080/$url"
+}
+
+# Make sure that Open Liberty is listening
+. "$dir/../../retry.sh" '_request / &> /dev/null'
+
+# Check that we can request /
+[ -n "$(_request '/')" ]
+
+# Check that the version.js file can be retrieved.
+helloWorld="$(_request '/version.js')"
+[[ "$helloWorld" == *'var current'* ]]
```

Comment:
IMO this test is barely better than testing that doing a `GET /` succeeds -- it'd probably be a good idea to improve it to do something slightly more meaningful at some point in the future (not worth blocking this PR for).

Reply:
Yeah, I probably should have started out by copying the jetty test rather than the tomcat one.